src/account_manager.rs

```diff
@@ -244,12 +311,14 @@ impl AccountManager {
                 backup_stronghold.account_get_by_id(&account_id_to_stronghold_record_id(account.id())?)?;
             let created_at_timestamp: u128 = account.created_at().timestamp().try_into().unwrap(); // safe to unwrap since it's > 0
             let stronghold_account = crate::with_stronghold_from_path(&self.storage_path, |stronghold| {
-                stronghold.account_import(
-                    Some(created_at_timestamp),
-                    Some(created_at_timestamp),
-                    stronghold_account.mnemonic().to_string(),
-                    Some("password"),
-                )
+                stronghold
+                    .account_import(
+                        Some(created_at_timestamp),
+                        Some(created_at_timestamp),
+                        stronghold_account.mnemonic().to_string(),
+                        Some("password"),
+                    )
+                    .map_err(|e| e.into())
```

Comment:
FYI you can replace single-method closures with the function directly (also means the compiler spends less time doing type inference) like `.map_err(Into::into)`. This also works with enum variants like `Ok`/`Some`/etc. because the compiler considers them to be functions rather than just type constructors.